docker-vertica/Dockerfile

```diff
@@ -138,9 +148,15 @@ RUN set -x \
   && /usr/sbin/useradd -r -m -s /bin/bash -g verticadba --uid ${DBADMIN_UID} dbadmin \
   # Allow passwordless sudo access from dbadmin
   && echo "dbadmin ALL=(ALL) NOPASSWD: ALL" | tee -a /etc/sudoers \
-  && echo "dbadmin -       nofile  65536" >> /etc/security/limits.conf 
-  
-ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
+  && echo "dbadmin -       nofile  65536" >> /etc/security/limits.conf \
+  # Set JAVA_HOME environment variable if not minimal, this will be loaded to all shells
+  && if [[ $MINIMAL != "YES" && $MINIMAL != "yes" ]]; then \
+    echo "JAVA_HOME=/usr" >> /etc/environment; \
+  fi \
+  # Create a symlink to python3 interpreter in vertica
+  && update-alternatives --install /usr/bin/python python /opt/vertica/oss/python3/bin/python3 1
```

Comment:
Why did we need to do this?  I removed this and was still able to run the python udx from /opt/vertica/sdk/examples.

Reply:
We don't need to do this.  Zhuoyu thought it would be a convenience if our python3 was in a normal path instead of needing to add /opt/vertica/oss/python3/bin/ to one's path.

Reply:
Zhuoyu reminded me that we needed this to run one of the sample C++ udx.  There is some code that invokes python to generate a bunch of numbers for the test. We want to build a CI that runs the udx tests, so adding this to the container made sense.